README.md

```diff
@@ -95,298 +55,23 @@ To run a Watcher only, first make sure you sent an ENV variable called with `INF
-C.f. [here](https://rinkeby.etherscan.io/tx/0x1a79fdfa310f91625d93e25139e15299b4ab272ae504c56b5798a018f6f4dc7b))
-
-we get
-```
-gas_reserve ~= (4 * 60 * 24 / 1) * 7 * 71505 * (40 / 10**9)  ~= 115 ETH
-```
-
-**NOTE** that the above calculation doesn't imply this is what is going to be used within a week, just a pessimistic scenario to calculate an adequate reserve.
-If one assumes an _average_ gas price of 4 Gwei, the amount is immediately reduced to ~11.5 ETH weekly.
-
-## Watcher
-
-The Watcher is an observing node that connects to Ethereum and the child chain server's API.
-It ensures that the child chain is valid and notifies otherwise.
-It exposes the information it gathers via an HTTP-RPC interface (driven by Phoenix).
-It provides a secure proxy to the child chain server's API and to Ethereum, ensuring that sensitive requests are only sent to a valid chain.
-
-For more on the responsibilities and design of the Watcher see [Tesuji Plasma Blockchain Design document](docs/tesuji_blockchain_design.md).
-
-### Modes of the watcher
-
-The watcher can be run in one of two modes:
-  - **security-critical only**
-    - intended to provide light-weight Watcher just to ensure security of funds deposited into the child chain
-    - this mode will store all of the data required for security-critical operations (exiting, challenging, etc.)
-    - it will not store data required for current and performant interacting with the child chain (spending, receiving tokens, etc.)
-    - it will not expose some endpoints related to current and performant interacting with the child chain (`account.get_utxos`, `transaction.*`, etc.)
-    - it will only require the `OMG.DB` key-value store database
-    - this mode will prune all security-related data not necessary anymore for security reasons (from `OMG.DB`)
-    - some requests to the API might be slow but must always work (called rarely in unhappy paths only, like mass exits)
-  - **security-critical and informational API**
-    - intended to provide convenient and performant API to the child chain data, on top of the security-related one
-    - this mode will provide/store everything the **security-critical** mode does
-    - this mode will store easily accessible register of all transactions _for a subset of addresses_ (currently, all addresses)
-    - this mode will leverage the Postgres-based `WatcherDB` database
-
-In releases, `watcher` refers to the security-critical mode, while `watcher_info` refers to the security-critical and informational API mode.
-
-### Using the watcher
-
-The watcher is listening on port `7434` by default.
-
-### Endpoints
-
-For API documentation see: https://omisego.github.io/elixir-omg
-
-### Ethereum private key management
-
-Watcher doesn't hold or manage user's keys.
-All signatures are assumed to be done outside.
-A planned exception may be to allow Watcher to sign challenges, but using a non-sensitive/low-funded Ethereum account.
-
-## `mix` configuration parameters
```

Comment:
I think this was a useful section, there are a lot of questions about these values being asked around. This is why it was added in the first place. Please consider keeping it and ping me if you need (or would like me to do it) help with refreshing it.

Reply:
Please re-check! :)